compiler/rustc_attr/src/builtin.rs

```diff
@@ -1285,3 +1285,29 @@ pub fn parse_confusables(attr: &Attribute) -> Option<Vec<Symbol>> {
 
     Some(candidates)
 }
+
+pub fn collect_doc_alias_symbol_from_attrs<'tcx>(
```

Comment:
Both calls to this function immediately use `contains(sym)` on the result, so it's probably better to pass `sym` to here and avoid the collection.